RPIPs/RPIP-43.md

```diff
@@ -15,6 +15,14 @@ tags: tokenomics-2024, tokenomics-content
 ## Abstract
 This proposal drastically reduces the gas to add validators and distribute rewards for them. It does so by creating a single contract that can be used as the Ethereum withdrawal address for any number of validators. In addition, the contract will facilitate the application of node-level penalties, as opposed to the current state where penalties are assigned and processed at the minipool level. Note that this proposal also enables ETH-only Node operation.
 
+## Motivation
+
+Megapools are motivated by several desires; the desire to lower ongoing costs, to lower the barrier to entry, and to support improved capital efficiency. Both ongoing and upfront costs are reduced by Megapools due to the efficiency savings achieved by using a single smart contract over multiple minipool contracts. Megapools support improved capital efficiency indirectly by allowing for a shared stake that can be seized if _any_ of an operator's validators misbehave, whereas, in the current system of disconnected stake-per-minipool, the stake can only be seized from the offending minipool.
```

Comment:
> Megapools allow lower initial and ongoing costs for node operators. Upfront costs are reduced by Megapools by using a single smart contract instead of multiple minipool contracts that need to be deployed separately. Ongoing costs are reduced by allowing consensus layer rewards to be distributed for many validators with a single transaction. These improvements are particularly important given the smaller bond sizes (and thus more validators per investment) proposed in [RPIP-42](RPIP-42).

It's possible to penalize across minipools as is (needs more accounting, but not necessarily a shared contract. The only capital efficiency improvements are via cost reduction, so focused more on that.